packages/@react-stately/table/src/TableCollection.ts

```diff
@@ -197,7 +219,7 @@ export class TableCollection<T> extends GridCollection<T> implements ITableColle
         }
       };
 
-      columns.unshift(rowHeaderColumn);
+      columns.push(rowHeaderColumn);
```

Comment:
why the change from unshift to push? that'll change its logical location in the table right?

Reply:
It just adds it after the drag button column. I supposed I could keep unshift, and just move the drag button column logic below it.